src/components/Dashboard/Rewards.tsx

```diff
@@ -145,17 +146,18 @@ export default function Rewards({
         <Statistic
           title={
             <SectionHeader
-              text={tokenSymbol ? tokenSymbol + ' tokens' : 'Tokens'}
-              tip={`${
-                tokenSymbol ? tokenSymbol + ' ERC20' : 'Tokens'
-              } are distributed to anyone who pays this project. If the project has set a funding target, tokens can be redeemed for a portion of the project's overflow whether or not they have been claimed yet.`}
+              text={tokenSymbol ? tokenSymbol + ' ' + t`tokens` : t`Tokens`}
+              tip={
+                `${tokenSymbol ? tokenSymbol + ' ' + t`ERC20` : t`Tokens`}` +
```

Comment:
**issue**: ideally we'd avoid splitting the sentence here for translation. Not sure if wrapping the whole thing in `t` will work though - would you mind giving it a crack?